lib/api/model/model.dart

```diff
@@ -781,6 +786,31 @@ enum ChannelPostPolicy {
     .map((key, value) => MapEntry(value, key));
 }
 
+/// Policy for whether a topic is required to post
+///
+/// For docs, search for "topics_policy"
+/// in <https://zulip.com/api/get-stream-by-id>
+@JsonEnum(valueField: 'apiValue')
+enum TopicsPolicy {
+  inherit(apiValue: "inherit"),
```

Comment:
From the API doc, I see that the meaning of `TopicsPolicy.inherit` depends on the organization-level `realm_topics_policy` value. So let's add that to our data store, in two separate commits before this:
- Add it to `InitialSnapshot`
- Add it to `RealmStore`, in the section "Realm settings found in realm/update_dict events"